support-models/src/main/scala/com/gu/support/zuora/api/SubscriptionData.scala

```diff
@@ -179,7 +179,8 @@ case class Subscription(
   readerType: ReaderType = ReaderType.Direct,
   promoCode: Option[PromoCode] = None,
   redemptionCode: Option[RawRedemptionCode] = None,
-  corporateAccountId: Option[String] = None
+  corporateAccountId: Option[String] = None,
+  giftNotificationEmailDate__c: Option[LocalDate] = None,
```

Comment:
Can we remove the `__c` from the end of this field and map it as we do with the other custom fields on this object here: https://github.com/guardian/support-frontend/blob/8cd4badd66b080321f84f79147459e1351ed48c0/support-models/src/main/scala/com/gu/support/zuora/api/SubscriptionData.scala#L165-L164

Reply:
I was wondering that, we should probably do the same for createdRequestId__c: String, on line 173 too.

Reply:
I've removed it from both now so it should be consistent.